.github/workflows/release.yml

```diff
@@ -1,22 +1,46 @@
-name: Release latest
+name: release
 
 on:
   push:
-    branches:
-      - main
+    tags:
+      - '*'
```

Comment:
Just in case, I'd suggest using `v**` instead. That way it's limited for v-prefixed semver tags.